scripts/hist_importer.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+# Copyright 2014-2017 Josefson Souza <josefson.br@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Tool to import browser history from other browsers."""
+
+
+import argparse
+import sqlite3
+import sys
+import os
+
+
+def parse():
+    """Parse command line arguments."""
+    description = ("This program is meant to extract browser history from your"
+                   "previous browser and import them into qutebrowser.")
+    epilog = ("Databases:\n\tQutebrowser: Is named 'history.sqlite' and can be"
+              " found at your --basedir. In order to find where your basedir"
+              " is you can run ':open qute:version' inside qutebrowser."
+              "\n\tFirerox: Is named 'places.sqlite', and can be found at your"
+              "system\"s profile folder. Check this link for where it is locat"
```

Comment:
I don't think you meant `system"s` here, but `system's`

Reply:
Might also want to move `located` to the next line instead of breaking in the middle of the word.